src/screens/Settings/SettingsPage.js

```diff
@@ -0,0 +1,79 @@
+// Setup
+import React, { Component } from "react";
+import { StyleSheet, View } from "react-native";
+
+// Vendor
+import {
+  Body,
+  Button,
+  CheckBox,
+  Container,
+  Content,
+  DatePicker,
+  Fab,
+  Form,
+  Header,
+  Icon,
+  Item,
+  Input,
+  Label,
+  Picker,
+  Text,
+  Title,
+  Toast
+} from "native-base";
+import { Formik } from "formik";
+
+// Redadalertas
+import { colors } from "styles";
+import eventServices from "services/event";
+import { checkIfLoggedIn } from "utils/user";
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "flex-start",
+  },
+});
+
+export default class SettingsPage extends Component {
+  static navigationOptions = () => ({ title: "Settings" });
+
+  onSubmit = async () => {
```

Comment:
It doesn't look like we use this function.

Reply:
This is a placeholder since I assumed we'd be making some basic user changes directly from the settings page (like username, location, etc)